src/locale/localed.c

```diff
@@ -715,31 +715,19 @@ static const sd_bus_vtable locale_vtable[] = {
         SD_BUS_PROPERTY("VConsoleKeymap", "s", property_get_vconsole, 0, SD_BUS_VTABLE_PROPERTY_EMITS_CHANGE),
         SD_BUS_PROPERTY("VConsoleKeymapToggle", "s", property_get_vconsole, 0, SD_BUS_VTABLE_PROPERTY_EMITS_CHANGE),
 
-        SD_BUS_METHOD_WITH_NAMES("SetLocale",
-                                 "asb",
-                                 SD_BUS_PARAM(locale)
-                                 SD_BUS_PARAM(interactive),
-                                 NULL,,
+        SD_BUS_METHOD_WITH_ARGS("SetLocale",
+                                 SD_BUS_ARGS("a", locale, "sb", interactive),
```

Comment:
fix the indentation, breaked out lines need to align to the open bracket - ie, remove one space from each line

Reply:
Also, the arguments are wrong - `locale` should be an array of strings (`as`(), and `interactive` should be a single boolean (`b`). See `org.freedesktop.locale1(5)` man page.

Reply:
Done sir.

Reply:
> Also, the arguments are wrong - `locale` should be an array of strings (`as`), and `interactive` should be a single boolean (`b`). See the `org.freedesktop.locale1(5)` man page.

I basically guessed that. I was hoping I'd be corrected.
Thank you sir.

Reply:
> Also, the arguments are wrong - `locale` should be an array of strings (`as`), and `interactive` should be a single boolean (`b`). See the `org.freedesktop.locale1(5)` man page.

Done sir.

Reply:
found this in the man page:
 methods:
      SetLocale(in  as locale,
                in  b interactive);
      SetVConsoleKeyboard(in  s keymap,
                          in  s keymap_toggle,
                          in  b convert,
                          in  b interactive);
      SetX11Keyboard(in  s layout,
                     in  s model,
                     in  s variant,
                     in  s options,
                     in  b convert,
                     in  b interactive);